domains/transfer/Transfer.API/API/ApiModels/Requests/RuleExtensions.cs

```diff
@@ -1,4 +1,9 @@
+using System;
+using System.Text;
+using API.Converters;
 using FluentValidation;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
```

Comment:
I do not think all these new using-statements are needed

Reply:
Fixed: https://github.com/Energinet-DataHub/energy-origin/pull/443/commits/af81e1579db39e8a4a95bf78dbfb95263d26410e